src/main/java/com/codeborne/selenide/impl/CiReportUrl.java

```diff
@@ -39,8 +41,23 @@ private String getTeamCityUrl() {
 
   private String getJenkinsReportsUrl() {
     String build_url = System.getProperty("BUILD_URL");
+    if (isEmpty(build_url)) {
+      build_url = System.getenv("BUILD_URL");
+    }
+    String workspace = System.getProperty("WORKSPACE");
+    if (isEmpty(workspace)) {
+      workspace = System.getenv("WORKSPACE");
+    }
```

Comment:
It can be simplified: `String workspace = System.getProperty("WORKSPACE", System.getenv("WORKSPACE"));`